attestation-service/docs/parsed_claims.md

```diff
@@ -93,6 +93,13 @@ The claim inherit the fields from the SEV-SNP claim with and additional `tpm` hi
 
 Note: The TD Report and TD Quote are fetched during early boot in this TEE. Kernel, Initrd and rootfs are measured into the vTPM's registers.
 
+## IBM Secure Execution (SE)
+- `se.version`: The version this quote structure.
+- `se.cuid`: The config uid.
+- `se.hdr.tag`: SE Header Tag (seht)
+- `se.image.phkh`: SE image Public host key hash
+- `se.attestation.phkh`: SE attestation Public host key hash
+- `se.user_data`: Custom attestation key owner data.
```

Comment:
Nice to see the parsed claims here. Need to add a new empty line here.

Reply:
Fixed, thanks!